pom.xml

```diff
@@ -88,6 +88,11 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>2.15.2</version>
```

Comment:
please use `<jackson.databind.version>2.13.4.2</jackson.databind.version>` 

to keep it consistent with Hydrator plugins.